### PR TITLE
update text

### PR DESCRIPTION
**Pull request by ealmdahl**
how you like them updated apples?

### CODE REVIEW

**Comment by sindrenm** [comment]
Hnnnnghghgh, I kinda like breaking lines in Markdown files. I know we've had this discussion somewhere earlier, but I can't remember the outcome.

File: HYPER_RECIPES.md [hunk 1]
```diff
@@ -6,12 +6,10 @@ Even though you could try and convince us to hire you for your good humor, charm
...
 
-We tend to enjoy some popcorn while reviewing your app, reading the code as if it was a fantasy book filled with dragons and unicorns. Please make it shine.
+If you want to have go at the recipes app (or we urge you to), we'll make you a nice and clean repo which you will fill with marvellous code. In our daily work, we work by submitting pull requests for all our features, updates, refactorings and bugfixes, and that's how we would like you to work, too. So start out by cloning the repo, create a new branch and start committing. Remember to keep the commits as small and specific as possible. When you have a feature done (or you want our comment on something), push it to github and create a pull request, so we can review and comment on it. We tend to enjoy some popcorn while reviewing your app, reading the code as if it was a fantasy book filled with dragons and unicorns. Please make it shine.
```
Comment (3lvis) [inline HYPER_RECIPES.md]: > So start out by cloning the repo, create a new branch and start committing.

I was thinking that they would get a clean repo just for their app, so cloning wouldn't be needed.

For me might be better if you remove that sentence :)

Comment (ealmdahl) [inline HYPER_RECIPES.md]: Well, wouldn't you start by running `git clone reponame`? But maybe "start out by creating a new branch" is better anyway? Or how would you formulate that sentence?

Comment (3lvis) [inline HYPER_RECIPES.md]: > In our daily work, we work by submitting pull requests for all our features, updates, refactorings and bugfixes, and that's how we would like you to work, too. Remember to keep the commits as small and specific as possible.

Something like this

Comment (3lvis) [inline HYPER_RECIPES.md]: I misread, you're correct they start by cloning the repo.

> So start out by cloning the repo, create a new branch and start submitting your pull requests!

**Comment by ealmdahl** [comment]
I see what you mean, @sindrenm. Should we agree on something there? Break after 80 characters?

**Comment by sindrenm** [comment]
That would make me a very happy camper, @ealmdahl! :grinning:

**Comment by ealmdahl** [comment]
Made it 100 characters wide. 80 felt like this
narrow

**Comment by sindrenm** [comment]
Love this. I'll merge it. :smiley:

**Comment by jgorset** [comment]
I think we concluded to break Markdown in that discussion, @sindrenm.

**Comment by jgorset** [comment]
Here: https://github.com/hyperoslo/playbook/pull/20

**Comment by sindrenm** [comment]
Right! Thanks for the link, @jgorset. Doesn't look like we concluded on an actual character count per line, though. The link I posted did mention 60 characters, but that's kinda … narrow. :stuck_out_tongue:

**Comment by jgorset** [comment]
90 or 100 feels fine to me.

**Comment by sindrenm** [comment]
:+1:

**Pull request merged**